lib/get-primary-value.js

```diff
@@ -7,6 +11,11 @@ module.exports = function getPrimaryValue (array) {
     return null;
   }
 
-  const primary = array.find((d) => d.primary);
```

Comment:
It's not, but we have a [polyfill](https://github.com/TheScienceMuseum/collectionsonline/blob/master/client/lib/polyfills.js#L18)